content/en/docs/concepts/configuration/secret.md

```diff
@@ -211,33 +220,32 @@ to create a Docker registry Secret, you can do:
 ```shell
 kubectl create secret docker-registry secret-tiger-docker \
   --docker-username=tiger \
-  --docker-password=pass113 \
-  --docker-email=tiger@acme.com \
-  --docker-server=my-registry.example:5000
```

Comment:
If it's important to include `--docker-server`, I can update the example to match it.

Reply:
Imo, I like keeping `--docker-password` , `--docker-email` , `--docker-server` as it's a comprehensive example.
They can also be found in the [pull images from a private registry task](https://kubernetes.io/docs/tasks/configure-pod-container/pull-image-private-registry/#create-a-secret-by-providing-credentials-on-the-command-line)

Reply:
I plan to update this.